.github/workflows/ci.yml

```diff
@@ -42,6 +42,14 @@ jobs:
           name: stats
           path: entrypoint-sizes.json
           if-no-files-found: error
+      - name: Export the test coverage report
+        run: nix-shell --run 'make ocaml-src && dune runtest --instrument-with bisect_ppx --force . && bisect-ppx-report summary --per-file | python scripts/coverage-to-json.py | tee test-coverage.json'
```

Comment:
It might be nicer to add this to the `Makefile`, eg. `make test-coverage.json` could depend on `ocaml-src` and run the commands we want (probably then the existing `test-coverage` task would just an alias for `test-coverage.json`).

Reply:
Ah yes, this command ended up being way too long. Currently our `test-coverage` target also generates the output as html, but I could have two targets, and share the common tasks. Then the one would finish with generating the html output and other would finish by generating json. I'll fix this :+1: